frodo.js

```diff
@@ -1,50 +1,45 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!